src/types/index.ts

```diff
@@ -24,6 +24,13 @@ export interface Config {
     ar: { title: string, body: string },
     duration: number
   },
+  downloadMohLandingPage: {
```

Comment:
I know it's a minor thing but please organize this block in the same way the other download blocks organized:
he
en
ru
ar
am